### PR TITLE
feat: add test event hubs message pump

### LINKED ISSUE #69: Add test version of Azure EventHubs message pump
**Is your feature request related to a problem? Please describe.**
Testing EventHubs message handlers is not always easy to test together as a fully working EventHubs resource is required for the message pump.

**Describe the solution you'd like**
Create a test version of an EventHubs message pump so that message handlers can be tested with ease.

### PR DESCRIPTION
**Pull request by stijnmoreels**
Adds a test version of an EventHubs message pump so that message handlers can be tested with ease.

Closes #69

### CODE REVIEW

**Review by fgheysels — COMMENTED** [review]
I started reviewing this, and then it seems like I forgot to finish it. :)

File: src/Arcus.Testing.Messaging.Pumps.EventHubs/TestAzureEventHubsMessageProducer.cs [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        /// Add a series of custom <paramref name="messages"/> which will result in <see cref="EventData"/> on the message pump.
+        /// </summary>
+        /// <typeparam name="TMessageBody">The custom type of the message body.</typeparam>
+        /// <param name="messages">The series of custom message bodies which will translate to <see cref="EventData"/> instances.</param>
+        /// <exception cref="ArgumentNullException">Thrown when the <paramref name="messages"/> is <c>null</c>.</exception>
+        public TestAzureEventHubsMessageProducer AddMessageBodies<TMessageBody>(IEnumerable<TMessageBody> messages)
+        {
+            Guard.NotNull(messages, nameof(messages), "Requires a series of message bodies to produce Azure Service Bus messages to the message pump");
+
+            EventData CreateServiceBusMessage(TMessageBody message)
```
[the remark this answers is not in the document]
Comment (stijnmoreels) [reply src/Arcus.Testing.Messaging.Pumps.EventHubs/TestAzureEventHubsMessageProducer.cs]: Woops. fixed in https://github.com/arcus-azure/arcus.testing/pull/70/commits/51c5f548daf625160356f43cf7307ab9445c4c2d

[the remark this answers is not in the document]
Comment (stijnmoreels) [reply src/Arcus.Testing.Messaging.Pumps.EventHubs/TestAzureEventHubsMessageProducer.cs]: Thx!

File: docs/preview/features/eventhubs-messsage-pump.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+        EventData message = 
+            EventDataBuilder.CreateForBody(BinaryData.FromObjectAsJson(reading))
+                            .Build();
+
+        return Task.FromResult(new[] { message });
+    }
+}
+```
+
+Such implementation can be pased along during the registration:
```
[the remark this answers is not in the document]
Comment (stijnmoreels) [reply docs/preview/features/eventhubs-messsage-pump.md]: Woops, fixed in https://github.com/arcus-azure/arcus.testing/pull/70/commits/b5a2b486888e72e0c01e3d0a6146b34f119621de.
Thx!

File: docs/preview/features/eventhubs-messsage-pump.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
+﻿---
+title: Testing Azure EventHubs message handling
+layout: default
+---
+
+# Testing Azure EventHubs message handling
+
+## Installation
+The following functionality is available when installing this package:
```
[the remark this answers is not in the document]
Comment (stijnmoreels) [reply docs/preview/features/eventhubs-messsage-pump.md]: Yeah, ok. Fixed in https://github.com/arcus-azure/arcus.testing/pull/70/commits/60cfdd9fe0a78268a40993153969227db029cbb5.

**Review by fgheysels — APPROVED** [review]

**Pull request merged**